Source/JavaScriptCore/llint/WebAssembly.asm

```diff
@@ -604,17 +604,29 @@ op(js_to_wasm_wrapper_entry, macro ()
 
     const AccumulatorTag = invalidGPR # Only used for JSVALUE32_64
 
+    # any callee saves used here would also have to be saved by the JS->WASM thunk,
+    # since exceptions cannot tell if an interpreted callee had a replacementCallee installed
+    # when the stack frame was created.
+    # We should just restore extra callee-saves before calling.
```

Comment:
What do you mean 'extra callee saves' -- above you say 'any callee saves', is there a subset of them which we wouldn't need to do that restore?

Reply:
Updated wording